administrator/language/de-DE/joomla.ini

```diff
@@ -349,8 +356,11 @@ JGLOBAL_BATCH_MOVE_ROW_NOT_FOUND="Die Zielzeile zum Verschieben wurde nicht gefu
 JGLOBAL_BATCH_PROCESS="Ausführen"
 JGLOBAL_BATCH_WORKFLOW_STATE_ROW_NOT_FOUND="Kann die betreffende Zeile für diese Statusänderung nicht finden."
 JGLOBAL_BLOG="Blog"
-JGLOBAL_BLOG_CLASS="CSS-Klasse Blog"
-JGLOBAL_BLOG_CLASS_LEADING="CSS-Klasse Blog (Führender Artikel)"
+JGLOBAL_BLOG_ACROSS_OPTION="Über"
+JGLOBAL_BLOG_CLASS="CSS-Klasse Beitrag"
```

Comment:
```suggestion
JGLOBAL_BLOG_CLASS="CSS-Klasse Blog"
```

Reply:
In #1691 wurde das im Core auf `JGLOBAL_BLOG_CLASS="Article Class"` geändert.

Andererseits ist in der `api//language/.../joomla.ini`  der gleiche String noch so zu finden:
`JGLOBAL_BLOG_CLASS="Blog Class"`

Reply:
> In #1691 wurde das im Core auf `JGLOBAL_BLOG_CLASS="Article Class"` geändert.
> 
> Andererseits ist in der `api//language/.../joomla.ini` der gleiche String noch so zu finden:
> `JGLOBAL_BLOG_CLASS="Blog Class"`

lassen wir erstmal so und schauen es uns dann in live noch mal an.
viele vergessen dabei leider immer auch die api anzupassen.

Reply:
Ich habe gerade nach Aufforderung von brianteeman einen PR https://github.com/joomla/joomla-cms/issues/34397 dazu geöffnet.
Die neuen Strings werden auch bei `/api` übernommen